src/components/AttachmentCarousel/CarouselActions/index.native.js

```diff
@@ -0,0 +1,85 @@
+import React, {Component} from 'react';
+import {PanResponder, Dimensions, Animated} from 'react-native';
+import PropTypes from 'prop-types';
+
+import styles from '../../../styles/styles';
+
+const propTypes = {
+    children: PropTypes.element.isRequired,
+
+    /** Callback to fire when swiping left or right */
+    onCycleThroughAttachments: PropTypes.func,
+
+    /** Callback to handle a press event because PanResponder will intercept all gestures */
+    onPress: PropTypes.func,
+
+    /** These help to prevent a swipe animation when at either end */
+    canSwipeLeft: PropTypes.bool,
+    canSwipeRight: PropTypes.bool,
```

Comment:
similar comments for onCycleThroughAttachments, onPress, canSwipeLeft and canSwipeRight. I don't think these should be optional since they are currently always passed.